src/client/cockpit-client.ui

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Generated with glade 3.38.2 -->
+<interface>
+  <requires lib="gtk+" version="3.24"/>
+  <requires lib="webkit2gtk" version="2.28"/>
+  <template class="CockpitClientWindow" parent="GtkApplicationWindow">
+    <property name="width-request">800</property>
+    <property name="height-request">600</property>
+    <property name="can-focus">False</property>
+    <property name="default-width">1280</property>
+    <property name="default-height">800</property>
+    <child>
+      <object class="WebKitWebView" type-func="webkit_web_view_get_type" id="webview">
+        <property name="visible">True</property>
+        <property name="can-focus">True</property>
+        <property name="is-ephemeral">True</property>
+        <child>
+          <placeholder/>
+        </child>
+      </object>
+    </child>
+    <child type="titlebar">
+      <object class="GtkHeaderBar" id="headerbar">
+        <property name="visible">True</property>
+        <property name="can-focus">False</property>
+        <property name="title" translatable="yes">Cockpit Client</property>
+        <property name="subtitle" translatable="yes" bind-source="webview" bind-property="title">Loading...</property>
```

Comment:
Please try to keep human visible strings out of this -- it otherwise needs i18n, which is probably too much effort for this single string.

Reply:
This slipped in via one of the broken-out PRs. I filed issue #16639 for this now.